pkg/config/helpers/general.go

```diff
@@ -0,0 +1,58 @@
+package helpers
+
+import (
+	"fmt"
+	"path/filepath"
+	"strings"
+)
+
+// ResolvePaths updates the given refs to be absolute paths, relative to the given base directory
+func ResolvePaths(refs []*string, base string) error {
+	for _, ref := range refs {
+		// Don't resolve empty paths
+		if len(*ref) > 0 {
+			// Don't resolve absolute paths
+			if !filepath.IsAbs(*ref) {
+				*ref = filepath.Join(base, *ref)
+			}
+		}
+	}
+	return nil
+}
+
+func MakeRelative(path, base string) (string, error) {
```

Comment:
@deads2k i know you will hate me, but I think if we call this "library" we should document the exported library functions ;-)

Reply:
> @deads2k i know you will hate me, but I think if we call this "library" we should document the exported library functions ;-)

sure